setup.py

```diff
@@ -54,39 +58,40 @@ def build_ply(self):
 
     def build_nt2(self):
         nt2_dir = 'nt2'
-        if not os.path.isdir(nt2_dir):
-            print('nt2 git repository not setup, cloning it')
-            cmd = 'git clone https://github.com/NumScale/nt2.git -b release'
-            check_call(cmd.split())
-        else:
-            cmd = 'git fetch && git checkout origin/release'
-            check_call(cmd, shell=True, cwd=nt2_dir)
-            print('nt2 git repository updated')
+        cwd = os.getcwd()
+        nt2_src_dir = os.path.join(cwd, self.build_temp, nt2_dir + '_src')
+        if not os.path.isdir(nt2_src_dir):
+            print('nt2 archive needed, downloading it')
+            url = 'https://github.com/NumScale/nt2/archive/release.zip'
```

Comment:
> And how do we handle updates for this zipfile?

We don't... Just ask for a new build, which is much faster now because
of faster download.